.circleci/config.yml

```diff
@@ -2,68 +2,32 @@
 version: 2.1
 orbs:
   commitlint: conventional-changelog/commitlint@1.0.0
-  
-executors:
-  default-executor:
-    docker:
-      - image: "cimg/node:10.22.0"
-    working_directory: ~/project
-    resource_class: medium
```

Comment:
I couldn't get this style working in combination with the Node orb's method (`node/default` with a tag) so I moved the executor specification into each job.